lib/themes/dosomething/paraneue_dosomething/scss/content/_finder.scss

```diff
@@ -213,7 +213,7 @@
   }
 
   .no-result {
-    background: $off-black asset-url("images/placeholder.png") center center;
+    background: $off-black center center;
```

Comment:
It [doesn't exist](https://github.com/DoSomething/phoenix/tree/dev/lib/themes/dosomething/paraneue_dosomething/images)! Since Webpack loads assets as well (to potentially inline as Data URIs), it was failing because the path didn't resolve to anything.